action.yml

```diff
@@ -40,8 +40,7 @@ inputs:
     default: arm64
   android-version:
     description: 'Android version.'
-    required: true
-    default: 12
```

Comment:
Please keep this line and set the default value to empty.

ref: https://github.com/dabao1955/kernel_build_action/blob/main/action.yml#L240